deploy/charts/istio-operator/Chart.yaml

```
@@ -9,7 +9,7 @@ sources:
 icon: https://istio.io/latest/img/istio-whitelogo-bluebackground-framed.svg
 
 # Based on support status of Istio releases: https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases
-kubeVersion: ">= 1.19.0-0 < 1.23.0-0"
+kubeVersion: ">= 1.20.0-0 <= 1.23.0-0"
 
-version: 2.0.8
-appVersion: "v2.12.0"
+version: 2.0.9
+appVersion: "v2.13.0"
```

When merged, I'd suggest to release a dev or rc release first to test, so sth like:
```suggestion
appVersion: "v2.13.0-rc.0"
```

Same for other places where this version number is used.